src/EnergyPlus/UnitarySystem.cc

```diff
@@ -2768,6 +2774,32 @@ namespace UnitarySystems {
                                                                              1,
                                                                              DataLoopNode::ObjectIsParent);
 
+                // these are needed for call from GetOASysNumHeat(Cool)ingCoils
+                std::string loc_heatingCoilType("");
+                if (fields.find("heating_coil_object_type") != fields.end()) { // not required field
+                    loc_heatingCoilType = UtilityRoutines::MakeUPPERCase(fields.at("heating_coil_object_type"));
+                    thisSys.m_HeatCoilExists = true;
+                }
+
+                std::string loc_coolingCoilType("");
+                if (fields.find("cooling_coil_object_type") != fields.end()) { // not required field
+                    loc_coolingCoilType = UtilityRoutines::MakeUPPERCase(fields.at("cooling_coil_object_type"));
+                    thisSys.m_CoolCoilExists = true;
+                }
+
+                // needed for checkUnitarySysCoilInOASysExists
+                if (UtilityRoutines::SameString(loc_coolingCoilType, "Coil:Cooling:DX:VariableSpeed")) {
+                    thisSys.m_CoolingCoilType_Num = DataHVACGlobals::Coil_CoolingAirToAirVariableSpeed;
+                } else if (UtilityRoutines::SameString(loc_coolingCoilType, "Coil:Cooling:WaterToAirHeatPump:VariableSpeedEquationFit")) {
+                    thisSys.m_CoolingCoilType_Num = DataHVACGlobals::Coil_CoolingWaterToAirHPVSEquationFit;
+                }
```

Comment:
Why only 2 possible types here?

Reply:
Look at line 15580 below. I needed to know at minimum if either of these 2 were the coil type or not. I could set coil type for all coils here but tried to be minimal in changes.